BuildResidentialHPXML/measure.rb

```diff
@@ -5828,6 +5901,35 @@ def self.set_hvac_distribution(hpxml_bldg, args)
     end
   end
 
+  def self.set_hvac_blower(hpxml_bldg, args)
+    # Blower fan W/cfm
+    hpxml_bldg.hvac_distributions.each do |hvac_distribution|
+      distribution_system_type = hvac_distribution.distribution_system_type
+
+      hvac_distribution.hvac_systems.each do |hvac_system|
+        next unless (distribution_system_type == HPXML::HVACDistributionTypeAir) || (hvac_system.is_a?(HPXML::HeatPump) && [HPXML::HVACTypeHeatPumpMiniSplit].include?(hvac_system.heat_pump_type))
```

Comment:
This logic won't work for a ductless MSHP since there is no distribution system. How about something like this?
```suggestion
    # Blower fan W/cfm
    hpxml_bldg.hvac_systems.each do |hvac_system|
        next unless (!hvac_system.distribution_system.nil? && hvac_system.distribution_system.distribution_system_type == HPXML::HVACDistributionTypeAir) || (hvac_system.is_a?(HPXML::HeatPump) && [HPXML::HVACTypeHeatPumpMiniSplit].include?(hvac_system.heat_pump_type))
```